docs/FAQ.md

```diff
@@ -21,33 +21,85 @@
 
 #### What is KUDO?
 
-KUDO is an operator we are building that is specifically designed to help provide operations to operators. We want to capture the actions that are required for managing applications in production as part of the definition for the applications themself. Further we want to embed those best practices in the operator CRD. So KUDO is the operator that handles those specs that are provided as declarative CRDs.
+Kubernetes Universal Declarative Operator (KUDO) provides a declarative approach to building production-grade Kubernetes [Operators](https://coreos.com/operators/).  It is an operator that is specifically designed to help provide operations to operators. We want to capture the actions that are required for managing applications in production as part of the definition for the applications themself. Further we want to embed those best practices in the operator [CRD](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+
+KUDO-based Operators don’t require any code in most cases, which significantly accelerates the development of Operators. It also eliminates sources of error and code duplication.
+
 
 #### When would you use KUDO?
 
 You would use KUDO when `kubectl apply -f` isn't quite enough to manage your application. If you are just applying the same YAML on updates you probably won't need the extra business logic KUDO gives you.
+Kudo should be used any time you would use an Operator.  It can provide an advanced user experience, automating such features as updates, backups and scaling.
 
 #### What is a Framework
 
-`Framework` is the highlevel CRD object. An example for a `Framework` is e.g. the [kafka-framework.yaml](https://github.com/kudobuilder/frameworks/blob/master/repo/stable/kafka/versions/0/kafka-framework.yaml) that you find in the [kudobuilder/frameworks](https://github.com/kudobuilder/frameworks) repo.
+`Framework` is the high-level description of a deployable service which is represented as an CRD object. An example `Framework` is the [kafka-framework.yaml](https://github.com/kudobuilder/frameworks/blob/master/repo/stable/kafka/versions/0/kafka-framework.yaml) that you find in the [kudobuilder/frameworks](https://github.com/kudobuilder/frameworks) repository.
+
+Kafka Framework Example
+```
+apiVersion: kudo.k8s.io/v1alpha1
+kind: Framework
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: kafka
+```
+
+More examples can be found in the https://github.com/kudobuilder/frameworks project and include: [flink](https://flink.apache.org/), [kafka](https://kafka.apache.org/), and [zookeeper](https://zookeeper.apache.org/).
+
 
 #### What is a FrameworkVersion?
 
-A `FrameworkVersion` is the particular implementation of a `Framework`. It contains: 
+A `FrameworkVersion` is the particular implementation of a `Framework` containing:
 
 - [Parameters](#what-is-a-parameter)
 - [Plans](#what-is-a-plan)
 - Kubernetes Objects
```

Comment:
Link to https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/ maybe?

Reply:
I initially thought the same! on it